.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "tabWidth": 2,
+  "useTabs": false,
+  "singleQuote": true
+}
```

Comment:
Might want to get rid of this

Reply:
I'm open to take this out but I think it gives value because it improves the Developer Experience for new comers (like me doing this PR) so I rather let @dolsem to decide :)

Reply:
Yeah let's remove this, since the library is already using eslint. This would create confusion.